fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingCommon.java

```diff
@@ -187,6 +190,16 @@ public long getLeaseLengthMillis() {
         return leaseLengthMillis;
     }
 
+    @Nonnull
+    public OnlineIndexer.ScrubbingPolicy getScrubbingPolicy() {
+        assert (scrubbingPolicy != null);// this can never happen, just eliminating the compiler warnings
```

Comment:
Nope. Only `IndexingScrubber` uses `getScrubbingPolicy`. And it can be created only when `common.isScrubber()' is true.

Reply:
I guess the claim your making is that everything outside of the `IndexingScrubber` calls `hasScrubbingPolicy` only? I guess that kind of makes sense, but it seems like it would make sense to validate that at run time to protect ourselves from writing bugs in the future. Or we move the `scrubbingPolicy` around so that we can't access it anywhere that it might be `null`.

Reply:
Ok. I've changed `common.getScrubbingPolicy` to Nullable, and created a local Nonnull `getScrubbingPolicy` in the scrubber module.